packages/core/src/core/createEvent.js

```diff
@@ -126,6 +129,14 @@ export default () => {
         event.mergeData(userData);
       }
 
+      if (content?.xdm?.timestamp) {
+        const enqueuedTime = new Date(content.xdm.timestamp).getTime();
+        let queueTimeMillis = Date.now() - enqueuedTime;
+        delete content.xdm.timestamp;
```

Comment:
Not every customer will want the timestamp completely gone. For this PR let's not delete the timestamp here, but allow customers to remove the timestamp in the onBeforeEventSend. I think it makes sense to make the timestamp context optional, and always pass queueTimeMillis, but that is for another ticket.